MultiYearStudies.R

```diff
@@ -1,44 +1,53 @@
 
-library(skimr)
 
-#duration
-meta_df %>%
-  group_by(Study_number, OtherFactor, Ecosystem_type) %>%
-  mutate(Duration = Study_midyear - min(Study_midyear)) -> meta_time
+#conduct meta-analysis
+#this is the best fitting model of irrigiation studies - not sure if the "/" notation is correct
+#does it makes sense to have Ecosystem_type as a random effect???? Maybe???
 
 
-#Not working at the moment
-meta_time %>%
-  select(-Study_number, -Record_number, -OtherFactor, -Manipulation_level, -Soil_type) %>%
-  skim() -> summary
 
+res_irrigation <- rma(yi, vi, random = ~ 1|Study_number/Ecosystem_type,
+              mods = ~Duration*Percent_control*Latitude,
+              method = "ML",
+           data = metadat[metadat$Manipulation == "Irrigation",],
+           test = "knha")
+summary.rma(res_irrigation)
+profile(res_irrigation)
+
+
```

Comment:
I comment here would probably be good: what's `rma.mv` doing?

Reply:
Done, but in the main file (rs-synthesis.Rmd).